corehq/apps/accounting/enterprise.py

```diff
@@ -119,7 +119,7 @@ def rows_for_domain(self, domain_obj):
             len(domain_obj.applications()),
             get_mobile_user_count(domain_obj.name, include_inactive=False),
             get_web_user_count(domain_obj.name, include_inactive=False),
-            sms_in_in_last(domain_obj.name, 30),
+            sms_in_last(domain_obj.name, 30),
```

Comment:
I'm guessing this was an autocomplete error, since it's hard to visually distinguish between `sms_in_in_last` and `sms_in_last` - I didn't actually notice that double `in` until jumping to the definition and seeing `direction="I"`.